### ISSUE TITLE
ComboBox loses focus outline when the dropdown menu is opened

### ISSUE THREAD

**Issue by vaadin-bot** [vaadin/framework#588]
**Originally by @jouni**

___
This is a usability defect. The focus outline should be visible as long as the user is using the component.

___
 Imported from https://dev.vaadin.com/ issue #2291

**Comment by asashour** [comment]
I can not reproduce this issue now, by looking into [sampler](https://demo.vaadin.com/sampler/#ui/data-input/multiple-value/combo-box)

**Comment by anasmi** [comment]
Looks like the issue has been solved and as @asashour has mentioned, I can't reproduce it either.

I will close this issue because of that, but if it's still remains for you, please, feel free to re-open this one or create a new one with example to reproduce. It will help us to narrow down the problem.

Thanks for your help and contributions! :)

**Comment by Legioth** [comment]
> feel free to re-open this one or create a new one with example to reproduce

Please note that non-maintainers cannot reopen issues that they haven't closed themselves. Typically, a comment with additional details will still trigger a maintainer to do the reopening.

**Comment by anasmi** [comment]
Thanks, @Legioth . I didn't know :)
